src/typeahead/typeahead.ts

```diff
@@ -91,6 +91,12 @@ export class NgbTypeahead implements ControlValueAccessor,
 
 
   /**
+   * A selector specifying the element the tooltip should be appended to.
+   * Currently only supports "body".
+   */
+  @Input() container: string;
```

Comment:
1. Can't we limit it to `@Input() container: 'body';` in this case?
2. You should probably add it to the `NgbTypeaheadConfig`